ps_newproducts.php

```diff
@@ -110,12 +110,13 @@ public function getContent()
         $output = '';
         if (Tools::isSubmit('submitBlockNewProducts')) {
             $productNbr = Tools::getValue('NEW_PRODUCTS_NBR');
+            $productNbDays = Tools::getValue('PS_NB_DAYS_NEW_PRODUCT');
 
             if (!$productNbr || empty($productNbr)) {
                 $output .= $this->displayError(
                     $this->trans('Please complete the "products to display" field.', [], 'Modules.Newproducts.Admin')
                 );
-            } elseif (0 === (int) $productNbr) {
+            } elseif (0 === (int) $productNbr || 0 === (int)$productNbDays) {
```

Comment:
```suggestion
            } elseif (0 === (int) $productNbr || 0 === (int) $productNbDays) {
```

Code is OK but PHP-CS-Fixer wants this little white space 😅